web/modules/custom/rdf_schema_field_validation/rdf_schema_field_validation.install

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Contains install and update hooks of the rdf_schema_field_validation module.
+ */
+
+use Drupal\rdf_entity\Entity\RdfEntityMapping;
+use EasyRdf\Graph;
+use EasyRdf\GraphStore;
+
+/**
+ * Implements hook_update_N().
+ */
+function rdf_schema_field_validation_update_8102() {
```

Comment:
This will never run. If you want to decouple that from the module (because the module should not be aware about that specific implementation) you should move it in core module.